docs-markdown/package.json

```diff
@@ -353,19 +347,24 @@
     ],
     "moduleFileExtensions": [
       "ts",
-      "js"
+      "tsx",

```

Comment:
I dont believe we need this file extension we should be able to remove this. jsx and tsx file extensions are for react files.

Reply:
Good catch, missed those.